src/WebJobs.Script/Rpc/LanguageWorkerState.cs

```diff
@@ -29,7 +29,7 @@ internal void DisposeAndRemoveChannel(ILanguageWorkerChannel channel)
         {
             if (_channels.TryRemove(channel.Id, out ILanguageWorkerChannel removedChannel))
             {
-                channel?.Dispose();
+                removedChannel.Dispose();
```

Comment:
You removed the `?` here... should it be there?

Reply:
thanks! good catch. Fixing now